download/download.adoc

```diff
@@ -40,25 +41,7 @@ The jBPM binaries include documentation, examples and sources.
 
 * Older community releases: Check https://download.jboss.org/jbpm/release/[the release archive].
 
-* Evaluation version of JBoss BPM Suite: Check http://www.jboss.com/downloads/[the product download site].
-
-=== Maven, Gradle, Ivy or Buildr
-
-The jBPM jars are also available in http://search.maven.org/#search|ga|1|org.jbpm[the central maven repository]
-(and also in https://repository.jboss.org/nexus/index.html#nexus-search;gav\~org.jbpm~\~~~[the JBoss maven repository]).
-
-If you use Maven, just add a dependency in your `pom.xml`:
-
-[source,xml]
-----
-    <dependency>
-      <groupId>org.jbpm</groupId>
-      <artifactId>jbpm-test</artifactId>
-      <version>#{site.pom.latestFinal.version}</version>
-    </dependency>
-----
-
-This is similar for Gradle, Ivy and Buildr.
+* Evaluation version of Red Hat Process Automation Manager: Check http://www.jboss.com/downloads/[the product download site].
```

Comment:
http://www.jboss.com/downloads is redirected to https://access.redhat.com/downloads - is there a reason for not putting the target URL there directly?

Reply:
frankly speaking not sure as it was already there, I just changed the BPM Suite to PAM